pandas/tests/groupby/test_transform.py

```diff
@@ -1074,3 +1076,58 @@ def test_transform_lambda_with_datetimetz():
         name="time",
     )
     assert_series_equal(result, expected)
+
+
+def test_transform_cumcount_ngroup():
```

Comment:
Is this not already covered by test below? Ideally we can rely on fixtures rather than one-off tests like this

Reply:
it's a bit different than below... the test below is just to ensure after `transform`, the index of transformed result is the same as the original dataset.

But you are right, this should not be in one-off tests... will look for if there is fixture for this already @WillAyd